### PR TITLE
New Deck of cartoons is added

### PR DESCRIPTION
**Pull request by AyushiGautam9**
I added a deck that is related to our childhood cartoons like Tom and Jerry etc . Hope you like it :)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (kat-kan) [inline .vscode/settings.json:2]: Can you please explain why did you need to add this line? Should it be merged into main?

File: decks.js [hunk 2]
```diff
@@ -1,1194 +1,1293 @@
...
+      cards: [
+        {
+          name: "Frozen",
+          content: {
+            primary: "❄️",
+            secondary: "🏔️🚪",
+          },
+          hint: "Has 2 female protagonists",
+          matchValue: "Arendelle",
+          seenHint: false,
```
Comment (kat-kan) [inline decks.js:16]: Please note that you modified the formatting across the whole file. Can you please limit the changes to only adding new deck?

File: decks.js [hunk 3]
```diff
@@ -1,1194 +1,1293 @@
...
-          {
-              name: "Shazam",
-              content: {
-                  secondary: "🧙‍♂️🌩"
+          ]
+    },
+    {
+      name: "Cartoons",
+          author: "Ayushi Gautam",
+          instructions: "The emojis on each card represent your favourite childhood cartoons you loved!. You can flip(click) the card and see a hint.Drag and drop a card onto the matching text clue on the right, which is cartoon's name in easy mode and some info regarding cartoon in challengening mode.",
```
Comment (kat-kan) [inline decks.js:1180]: "some info" is not very specific. Would it be possible to narrow it somehow? For example in the already existing decks you can see that challenging mode uses character names or places.

**Review by kat-kan — COMMENTED** [review]
Hi @N-Shar-ma I wanted to help a bit and reviewed new code, hope you don't mind :)

**Comment by kat-kan** [comment]
@AyushiGautam9 just to be sure, have you read contribution guideline? [here](CONTRIBUTING.md)
Please note that it's required to work on a branch, not master
Have you tested your changes?

File: decks.js (hunk 3 above)
Comment (AyushiGautam9) [reply decks.js:1180]: yeah sure, I'll try to do that

**Comment by AyushiGautam9** [comment]
> @AyushiGautam9 just to be sure, have you read contribution guideline? [here](CONTRIBUTING.md) Please note that it's required to work on a branch, not master Have you tested your changes?

@kat-kan I guess I missed that point , sorry  and I'll take care of that

File: decks.js (hunk 3 above)
Comment (N-Shar-ma) [reply decks.js:1180]: The punctuation is a bit messed up here (`!` followed by `.` and no space after `.`) so please fix that as well

**Comment by N-Shar-ma** [comment]
@kat-kan Really appreciate the reviews! Thanks a bunch! 😄

**Comment by N-Shar-ma** [comment]
@AyushiGautam9 The cards looks good! Please address the feedback left, and then this should probably be good to merge!
